examples/pytorch/language-modeling/run_clm.py

```diff
@@ -572,6 +574,13 @@ def compute_metrics(eval_preds):
         if training_args.do_eval and not is_torch_tpu_available()
         else None,
     )
+    # Mlflow initial
+    experiment_name =f'language-modeling-clm-{model_args.model_name_or_path}'
+    #set the os enviroment for MLflowCallback
+    os.environ["DISABLE_MLFLOW_INTEGRATION"] = "False"
+    os.environ["MLFLOW_EXPERIMENT_NAME"]=experiment_name
```

Comment:
Must these variables be set in this training file or can also be set in the env?

Reply:
Setting envvar in a python script looks very awkward here. I suggest either:
* moving them to the shell script 
* using equivalent python API from `mlflow`. For example, instead of setting `MLFLOW_EXPERINMENT_NAME`, explicitly call `mlflow.create_experiment()` / `mlflow.get_experiment()` / `mlflow.get_experiment_by_name()`

Reply:
The HF have a callback for MLFlow (you can check it out here: https://huggingface.co/docs/transformers/main_classes/callback#transformers.integrations.MLflowCallback), and it take value from these env var. I think that we should use this and we don't have to create mlflow code by ourselves.

Reply:
For @huonglarne  questions, I think these env var should stay in the train script because it will change depending on your task and model. But some variables with bool values can be set in the env file